docs/content/node-operation/access-node-setup.mdx

```diff
@@ -1,22 +1,328 @@
 ---
 title: Setting Up a Flow Access Node
 sidebar_title: Access Node Setup
-description: How to run a Permissionless Access node
+description: How to run a permissionless Access node
 ---
 
-This guide is for running a Permissonless Access node on Flow. If you are planning to run a different type of staked node then see [node bootstrap](node-bootstrap.mdx).
+This guide is for running a permissonless Access node on Flow. If you are planning to run a different type of staked node then see [node bootstrap](node-bootstrap.mdx).
+
+Permissionless Access nodes allow any operator to run a Flow Access node.
+Unlike the other staked nodes, a permissionless access node does not have to be approved by the service account before it can join the network, hence the term "permissionless". The goal is to make all node type permissionless and this is the first step towards achieving that goal.
+
+## Who should run a Permissionless Access node?
+dApp developers can choose to run their own private permissionless access node and move away from using the community access nodes. This will also allow them to be not subjected to the API rate limits of the shared access nodes.
+
+Node operators can also run their own permissionless access node and provide access to that node as a service.
+
+Chain analytics, audit and exploration applications can run such an access node and do not have to rely on third parties for the state of the network.
+
+## Limitations
+This is a preliminary implementation of full permissionless node operation and there will be exactly **five** slots in total that will be opened for access nodes.
+If there are more than five staked access nodes at the end of the staking phase on 14th Feb 2023 EOD PT, five of those will be selected by an on-chain random selection process and no other permissionless ANs can join until others unstake in the next epoch.
+
+More slots will be opened in the future if this goes well.
+
+Support for unlimited access nodes requires staking auctions to be implemented, which is still an upgrade planned for the future.
+
+## How to run a Permissionless Access node?
+
+> Note: To run an access node you will need to provision a machine or virtual machine to run your node software. Please follow the [node-provisioning](node-provisioning.mdx) guide for it.
+You can provision the machine before or after your node has been chosen.
+
+At a high level, to run a permissionless Access node, you will have to do the following steps:
+1. Generate the node identity (private and public keys, node ID etc.).
+2. Stake the node with 100 FLOW by **EOD Tuesday, 14th February 2023 Pacific time (02/14/2023 12:00 PM Pacific or 02/15/2023 8:00 AM UTC)** by providing the node information generated in step 1.
```

Comment:
Is this supposed to be `12:00 AM`?

Reply:
Maybe add a note to see the `Timing` section for details

Reply:
just changed it to 23:50 to avoid confusion